### PR TITLE
Add VPN gateway docs, small clean up for external IPs docs

### PR DESCRIPTION
**Pull request by SeventhOfNine**
Add VPN gateway docs:

- List
- Retrieve
- Create
- Delete

### CODE REVIEW

**Review by JamieWulala — APPROVED** [review]
LGTM, just make sure the conflicts are resolved

File: source/includes/gcp/_vpn_gateways.md [hunk 1]
```diff
@@ -1 +1,186 @@
...
+    "externalIp": {},
+    "tunnelsName": [],
+    "reserveStaticIP": false,
+    "id": "3112849056897469664",
+    "name": "test-vpn-gw",
+    "shortRegion": "us-central1"
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/vpngateways/:id</code>
```
Comment (franzpgarcia) [inline source/includes/gcp/_vpn_gateways.md:101]: seeing weird error in github but looks to be working.

Comment (SeventhOfNine) [reply source/includes/gcp/_vpn_gateways.md:101]: I'll check the linting, could be spaces or weirdo stuff. Thanks.

Comment (SeventhOfNine) [reply source/includes/gcp/_vpn_gateways.md:101]: Change pushed.

**Pull request merged**